nototools/generate_website_2_data.py

```diff
@@ -754,30 +740,37 @@ def copy_font(self, fontpath):
 
   def build_family_css(self, key, family):
     fonts = [m for m in (family.hinted_members or family.unhinted_members)
-             if m.variant != 'Mono' and not m.is_UI]
-    fonts.sort(key=lambda f: str(css_weight(f.weight)) + '-' +
-               ('b' if css_style(f.slope) == 'italic' else 'a'))
+             if not m.is_UI]
+    fonts.sort(
+        key=lambda f: (
+            f.is_mono, css_weight(f.weight), css_style(f.slope) == 'italic'))
 
     css_name = key + '.css'
-    csspath = path.join(self.css, css_name)
-    font_size = 0
-    with open(csspath, 'w') as css_file:
+    css_path = path.join(self.css, css_name)
+    max_font_size = 0
+    with open(css_path, 'w') as css_file:
       for font in fonts:
-        font_name = self.copy_font(font.filepath)
-        font_size = max(font_size, os.stat(font.filepath).st_size)
+        font_path = self.copy_font(font.filepath)
+        max_font_size = max(max_font_size, os.stat(font.filepath).st_size)
+        # Make it possible to access Mono cjk variants and those with irregular
+        # css values by assigning them other names.
+        css_family = family.name
+        if font.is_cjk and font.is_mono:
+          css_family += ' ' + 'Mono'
+        weight = css_weight(font.weight)
+        if weight % 100 != 0:
+          css_family += ' ' + str(weight)
+          # prevent auto-bolding of this font by describing it as bold
+          weight = 700
+        slope = css_style(font.slope)
         css_file.write(
           '@font-face {\n'
           '  font-family: "%s";\n'
           '  font-weight: %d;\n'
           '  font-style: %s;\n'
           '  src: url(../fonts/%s) format("truetype");\n'
```

Comment:
I don't what browsers would do with format set to 'truetype' for CFF fonts. 

BTW, where is this CSS used? Internal only or for get/noto page?  If it's the latter, shouldn't we have woff2 and woff compressed versions instead of otf/ttf?